src/core/components/hv-navigator/index.tsx

```diff
@@ -129,38 +132,34 @@ export default class HvNavigator extends PureComponent<Props> {
           screens.push(buildTabScreen(id, type));
         }
       }
-    }
+    });
 
-    // Add the dynamic screens
-    switch (type) {
-      case NavigatorService.NAVIGATOR_TYPE.STACK:
-        // Dynamic is used to display all routes in stack which are presented as cards
-        screens.push(
-          <Stack.Screen
-            key={NavigatorService.ID_DYNAMIC}
-            component={HvRoute}
-            getId={({ params }) => params?.url}
-            // empty object required because hv-screen doesn't check for undefined param
-            initialParams={{}}
-            name={NavigatorService.ID_DYNAMIC}
-          />,
-        );
-
-        // Modal is used to display all routes in stack which are presented as modals
-        screens.push(
```

Comment:
I had commented about this in your previous PR, it look like we were missing a "case" statement here - the refactor doesn't seem to fix that

Reply:
Both screens are required for stack navigator, so both are added. No case statement needed here.